Makefile

```diff
@@ -26,6 +26,11 @@ cover:
 	go test -race -covermode=atomic -coverprofile=coverage.out ./...
 .PHONY: cover
 
+mock:
```

Comment:
That's great! :+1: Should we document how to use it somewhere? Maybe a new `development` section in README (or a separate doc file) to accommodate information about how to generate mocks, write and run the tests, etc.

Reply:
I agree. I don't want the landing readme to get too bloated. I feel like it's aleady pushing it. We could add it to [CONTRIBUTING.md](https://github.com/nikoksr/notify/blob/main/CONTRIBUTING.md) tho.